### ISSUE TITLE
Answers SearchBar Borderless on Mobile

### ISSUE THREAD

**Issue by benmcginnis** [yext/answers-search-ui#624]
<img width="369" alt="Screen Shot 2020-03-31 at 11 19 15 PM" src="https://user-images.githubusercontent.com/1206953/78095871-17d7b780-73a6-11ea-9a9a-2655ea4d7d6a.png">

Caused by [this](https://github.com/yext/answers/blob/69a33d0628c560e13626d9ad9d7ad98bbeeae02b/src/ui/sass/modules/_SearchBar.scss#L42) line being in a media query

**Comment by rosiegrant** [comment]
this is intentional styling from jeremy

**Comment by benmcginnis** [comment]
I don’t think it’s fully thought through.  It doesn’t work on a white background. I think if we’re going to do more heavy customizations of the experience going forward, we should consider separating the default answers experience into a theme that’s separate from the base configuration of the components, which would be less tied to the styling of the default theme.

**Comment by rosiegrant** [comment]
point taken ☺️ 

For whomever works on this, I think we can just remove the media query restriction here, 

https://github.com/yext/answers/blob/13ea6669234ed61d201f01f62326d01bb0aa6320/src/ui/sass/modules/_SearchBar.scss#L40

Here's what the HHs have been adding (for reference)
```
.yxt-SearchBar
{
  &-container
  {
    @include bplte(sm) {
      border-radius: .4rem;
      border: 1px solid $gray-3;
    }
  }
}
```

**Comment by rosiegrant** [comment]
edit: Jeremy is going to look at this this week and will provide some recommendations for the default UX

**Comment by benmcginnis** [comment]
We should definitely make sure he considers the search bar independently of
its context, in addition to the whole default layout.

**Comment by rosiegrant** [comment]
@bryanreed let's move forward keeping the search border the same on desktop and mobile. thanks!

**Comment by rosiegrant** [comment]
resolved as of 1.0.0
